wstesting/wstesting.go

```diff
@@ -305,7 +293,9 @@ func (cmd cSet) run(ctx *Context) error {
 		return fmt.Errorf("worksheet %s not yet created", cmd.ws)
 	}
 	for field, value := range cmd.values {
-		if err := ws.Set(field, value); err != nil {
+		if v, err := value.eval(ctx); err != nil {
+			return err
+		} else if err := ws.Set(field, v); err != nil {
```

Comment:
interesting. i had thought of these assignments as scoped to the branch, didn't know there was shared local scope for all branches in an if.